src/app/components/DataLoader/DataLoader.jsx

```diff
@@ -1,10 +1,73 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import Actions from '@Actions';
+import { ajaxCall } from '@utils';
+import appConfig from '@appConfig';
 
 class DataLoader extends React.Component {
+  constructor(props) {
+    super(props);
+    const { location } = this.props;
```

Comment:
You're not using `location` in the constructor so you don't need this.

Reply:
Removed